test/integration/test_kraken.py

```diff
@@ -117,5 +154,34 @@ def setUpClass(cls):
         cls.fastqs = cls.input_fastqs()
 
 
+class TestKrakenKrona(TestKrakenBase, TestKronaBase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestKrakenKrona, cls).setUpClass()
+        TestKronaBase.setUpClass()
+        cls.data_dir = join(util.file.get_test_input_path(), 'TestKrakenViralMix')
+        cls.db_dir = os.path.join(cls.data_dir, 'db')
+        cls.db = cls.build_kraken_db()
+        cls.krona_db = cls.build_krona_db()
+        cls.bam = join(cls.data_dir, 'test-reads.bam')
+        cls.fastqs = cls.input_fastqs()
+
+    def test_kraken_krona(self):
+        bin = join(util.file.get_project_path(), 'metagenomics.py')
+        out_report = util.file.mkstempfname('.report')
+        out_reads = util.file.mkstempfname('.reads.gz')
+        cmd = [bin, 'kraken', self.bam, self.db, '--outReport', out_report, '--outReads', out_reads]
+        util.misc.run_and_print(cmd, check=True)
+        out_html = util.file.mkstempfname('.krona.html')
+        cmd = [bin, 'krona', out_reads, out_html, '--db', self.krona_db]
+        print(cmd)
+        try:
+            util.misc.run_and_print(cmd, check=True)
```

Comment:
I've been realizing one big drawback to the way you're testing commands (by actually spawning a whole python subprocess in order to test argparse and everything) is that it means none of it shows up in coveralls at all.

The way most of our other unit tests do this would look like this:
`
 parser = metagenomics.parser_krona(argparse.ArgumentParser())
 args = parser.parse_args([out_reads, out_html, '--db', self.krona_db])
 args.func_main(args)
`

Can you switch these invocations of run_and_print over?